process.c

```diff
@@ -4353,6 +4353,12 @@ rb_call_proc__fork(void)
  *  This method is not for casual code but for application monitoring
  *  libraries. You can add custom code before and after fork events
  *  by overriding this method.
+ *
+ *  Note: Process.daemon may be implemented using fork(2) in Linux and
+ *  macOS (and possibly others) BUT does not go through this method.
```

Comment:
I doubt that the specific system names are necessary, even except that it depends on libc and not the kernel.

Reply:
Got it! I've updated it to mention the libc instead:

> Note: Process.daemon may be implemented using fork(2) in glibc (Linux), libSystem (macOS), and possibly others BUT does not go through this method.

let me know if you'd prefer that I just remove the references to Linux and macOS from here 😄

Reply:
Well, I think “using fork(2)” is just enough.

Reply:
Got it! Updated with your guidance :)